dev-support/mini-submarine/submarine/run_customized_submarine-all_mnist.sh

```diff
@@ -49,7 +49,11 @@ else
   WORKER_CMD="myvenv.zip/venv/bin/python mnist_distributed.py --steps 2 --data_dir /tmp/data --working_dir /tmp/mode"
 fi
 
-SUBMARINE_VERSION=0.3.0-SNAPSHOT
+SUBMARINE_VERSION=$SUBMARINE_VER
```

Comment:
SUBMARINE_VERSION =${SUBMARINE_VER:-"0.3.0-SNAPSHOT"}
This way, So you can set default values.

Reply:
Yeah. Will simplify it.